examples/keycloak-config-cli-using-spire/spire-values.yaml

```diff
@@ -0,0 +1,35 @@
+spiffe-oidc-discovery-provider:
+  tools:
+    kubectl:
+      image:
+        tag: v1.26.10 # Only required if the rancher/kubectl image doesn't exist for your k8s version
```

Comment:
Likely the example will not be used on this version. Lets remove it here and make the chart easier to override for this. Issue here:
https://github.com/spiffe/helm-charts-hardened/issues/250
```suggestion
```

Reply:
Removed. I added a pinned node version for the cluster creation for now. The `rancher/kubectl:v1.29.1` doesn't exist yet.